### ISSUE TITLE
[A11y] Work Show Page-All - Tab list

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2011]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n
https://staging.oregondigital.org/concern/images/1g05fb620

The tabs below the image viewer ("DETAILS", "HOW TO CITE", and "CONTACT US") are reporting that they're a tab list, but they are not coded correctly to function as such a widget.

### Suggested Fix

A proper tablist needs to follow all the required implementation details at https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel

- There is a lot of keyboard functionality these tabs lack, such as being a single "widget" when tabbing over links, or allowing arrow keys to move right and left across tabs.
- The tabs are sort of behaving like disclosures when you select them; the tab widget on the W3C page tells you a tab is selected when you arrow to a new tab, but OD is saying "expanded" when you activate one of the tab links. Since there is no collapse state this is confusing.

This could be implemented using a disclosure or accordion patterns. They're somewhat less complex in terms of keyboard support, but they could be a pretty significant reworking of the HTML.

### Can anybody QA or does this need an expert?

Needs expert

### QA Steps

Depends on fix - verify tabs are implemented per the spec, or that the area is fixed per whatever other spec might be chosen
